Bio/SeqIO/InsdcIO.py

```diff
@@ -450,7 +450,7 @@ class GenBankWriter(_InsdcWriter):
     SEQUENCE_INDENT = 9
 
     def _write_single_line(self, tag, text):
-        """Used in the 'header' of each GenBank record."""
+        """Write single line in each GenBank record (PRIVATE)."""
```

Comment:
Can you include the original information about where this is used (see comment on next example),